### PR TITLE
Poll

### PR DESCRIPTION
**Pull request by cyklon73**
## Checklist

 - [X] Read [Contribution Guidelines](https://github.com/SlimeCloud/java-SlimeBot#-contributing) and [Style Guides](https://github.com/SlimeCloud/java-SlimeBot#-style-guide)
 - [x] Test the Code
 - [x] This PR is ready to review and merge

## Description
Add poll system

## Related Issue
N/A

## Other Information
N/A

### CODE REVIEW

File: src/main/java/com/slimebot/util/Util.java [hunk 1]
```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
...
+
+	public static String padRight(String s, int n) {
+		return padRight(s, ' ', n);
+	}
+
+	public static String padLeft(String s, int n) {
+		return padLeft(s, ' ', n);
+	}
+
+	public static String padRight(String s, char padChar, int n) {
```
Comment (MineKing9534) [inline src/main/java/com/slimebot/util/Util.java:63]: Make padChar String

File: src/main/java/com/slimebot/util/Util.java [hunk 2]
```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
...
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.append(padChar);
+		return sb.toString();
```
Comment (MineKing9534) [inline src/main/java/com/slimebot/util/Util.java:66]: return s + padString,repeat(n);

File: src/main/java/com/slimebot/util/Util.java [hunk 3]
```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
...
+	}
+
+	public static String padLeft(String s, char padChar, int n) {
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.insert(0, padChar);
+		return sb.toString();
```
Comment (MineKing9534) [inline src/main/java/com/slimebot/util/Util.java:72]: return padString.repeat(n) + s

**Review by MineKing9534 — CHANGES_REQUESTED** [review]

File: src/main/java/com/slimebot/util/Util.java (hunk 1 above)
Comment (cyklon73) [reply src/main/java/com/slimebot/util/Util.java:59]: why? it must be a char that can be perfectly adapted to the specified length

File: src/main/java/com/slimebot/util/Util.java (hunk 2 above)
Comment (cyklon73) [reply src/main/java/com/slimebot/util/Util.java:62]: no. you have not understood the meaning behind pad. not a certain number of characters should be added, the specified character should be appended until the string has reached a certain length. so if String s is longer, less characters will be added.

File: src/main/java/com/slimebot/util/Util.java [hunk 4]
```diff
@@ -42,4 +47,51 @@ public static boolean isValidURL(String url) {
...
+		return padLeft(s, ' ', n);
+	}
+
+	public static String padRight(String s, char padChar, int n) {
+		StringBuilder sb = new StringBuilder(s);
+		while (sb.length()<n) sb.append(padChar);
+		return sb.toString();
+	}
+
+	public static String padLeft(String s, char padChar, int n) {
```
[the remark this answers is not in the document]
Comment (cyklon73) [reply src/main/java/com/slimebot/util/Util.java:65]: what?

File: src/main/java/com/slimebot/util/Util.java (hunk 3 above)
Comment (cyklon73) [reply src/main/java/com/slimebot/util/Util.java:68]: no, for the same reason as in my other comment

File: src/main/java/com/slimebot/util/Util.java (hunk 2 above)
Comment (MineKing9534) [reply src/main/java/com/slimebot/util/Util.java:62]: Still:
```java
public static String padRight(String s, chat padChar, int n) {
  return s + String.valueOf(padChar).repeat(n - s.length());
}
```

**Review by MineKing9534 — APPROVED** [review]

**Review by MatitoDA — APPROVED** [review]

**Pull request merged**